src/server/game/Entities/Unit/StatSystem.cpp

```diff
@@ -1327,6 +1327,11 @@ void Guardian::UpdateAttackPowerAndDamage(bool ranged)
             }
 
             bonusAP = owner->GetTotalAttackPowerValue(RANGED_ATTACK) * 0.22f * mod;
+            if (AuraEffect* aurEff = owner->GetAuraEffectOfRankedSpell(34453, EFFECT_1, owner->GetGUID())) // Animal Handler
+            {
+                AddPct(bonusAP, aurEff->GetAmount());
+                AddPct(val, aurEff->GetAmount());
```

Comment:
Thanks for the explanation. Looking at that code again, I wonder if the sequence of those 2 lines should be swapped, but maybe the order is indifferent like when performing a regular addition?